api/src/main/java/org/openmrs/Auditable.java

```diff
@@ -19,49 +17,9 @@
  * this interface.
  * 
  * @since 1.5
- * @see OpenmrsData
- * @see OpenmrsMetadata
+ * @see MutableOpenmrsData
+ * @see MutableOpenmrsMetadata
  */
-public interface Auditable extends OpenmrsObject {
-	
-	/**
-	 * @return User - the user who created the object
-	 */
-	public User getCreator();
-	
-	/**
-	 * @param creator - the user who created the object
-	 */
-	public void setCreator(User creator);
-	
-	/**
-	 * @return Date - the date the object was created
-	 */
-	public Date getDateCreated();
-	
-	/**
-	 * @param dateCreated - the date the object was created
-	 */
-	public void setDateCreated(Date dateCreated);
-	
-	/**
-	 * @return User - the user who last changed the object
-	 */
-	public User getChangedBy();
-	
-	/**
-	 * @param changedBy - the user who last changed the object
-	 */
-	public void setChangedBy(User changedBy);
-	
-	/**
-	 * @return Date - the date the object was last changed
-	 */
-	public Date getDateChanged();
-	
-	/**
-	 * @param dateChanged - the date the object was last changed
-	 */
-	public void setDateChanged(Date dateChanged);
-	
+public interface Auditable extends Creatable, Changeable {
```

Comment:
Should we actually get rid of this interface? Or should we keep it around?

Reply:
Unless we can scan core code and popular modules and ensure nobody is referencing it, I would keep it around for now.

Reply:
Definitely it's referenced in core, but in the follow up commit I was going to replace its usage. I wasn't going to remove it in this version, we would mark it as deprecated now and eventually remove it may be 2 versions later.